feature-libs/organization/user-registration/components/verification-token-form/verification-token-form-component.service.ts

```diff
@@ -75,7 +75,9 @@ export class RegisterVerificationTokenFormComponentService {
     if (
       this.authConfigService.getOAuthFlow() ===
         OAuthFlow.ResourceOwnerPasswordFlow ||
-      this.featureConfigService.isEnabled('authorizationCodeFlowByDefault')
+      this.featureConfigService.isEnabled(
+        'nativeSupportForRedirectingOAuthFlows'
```

Comment:
This change is related to the enhancements we made to support oAuth flows involving a redirect.  We now want to always redirect to login, not only for ROPC grant.